src/python/pants/option/global_options.py

```diff
@@ -242,7 +242,7 @@ def register_bootstrap_options(cls, register):
             help="Regexps matching warning strings to ignore, e.g. "
             '["DEPRECATED: scope some_scope will be removed"]. The regexps will be matched '
             "from the start of the warning string, and will always be case-insensitive. "
-            "See the `warnings` module documentation for more background on these are used.",
+            "See the `warnings` Python module documentation for more background on how these are used.",
```

Comment:
Would be great to add this: https://docs.python.org/3/library/warnings.html

Also nit, this might be >100 lines. Black does not enforce line length if it's a string, so you have to manually fix it.